routes/web-user.php

```diff
@@ -19,4 +20,13 @@
 
 Route::get('/home', [HomeController::class, 'index'])->name('home');
 
-Auth::routes();
\ No newline at end of file
+Route::post('/insert/{user}', [TaskController::class, 'insert'] )->name('data.insert');
+
+Route::get('/show/{task}', [TaskController::class, 'show'] )->name('data.show');
+
+Route::put('/update/{task}', [TaskController::class, 'update'] )->name('data.update');
+
+Route::delete('/delete/{task}', [TaskController::class, 'delete'] )->name('data.delete');
+
+Auth::routes();
```

Comment:
- Put this entire route list inside the Route::middleware(['auth:web'])->group(function(){}) so only login users can access them.
- Try grouping the routes according to its resource:
- Example:
  Route::prefix("/tasks")->as('tasks.')->group(function(){
    Route::get('/', [TaskController::class, 'showAll'])->name('index');
     ...
  });

- Arrange the routes like this:
       /get requests 
       /post requests 
        /put requests 
      /delete requests

- For update/delete methods,no need to put /update, /delete as its prefix.